src/tasks.js

```diff
@@ -0,0 +1,195 @@
+import {
+  deleteArrOfIndex, editTask, exhangeData, getData, setData,
+} from './data.js';
+import createNewElement from './newElement.js';
+
+class Task {
+  constructor() {
+    document.querySelector('#refresh-btn').addEventListener('click', this.refreshTasks);
+    this.draggedLiId = null;
+  }
+
+  buildForm1 = () => {
```

Comment:
Yeah, thank you for your suggestion, can you give more detailed information for me to get the difference between using regular or arrow _functions_ inside a class.

Reply:
Thank you for your feedback, @khanh-devos. I'm glad to hear that my previous response was helpful to you. 🤝 

 

- [ ] As you mentioned, since you're not using bind() inside the constructor and the arrow functions are serving your needs effectively, you can continue to use arrow functions in your code.